DigitalAssetManagement/README.md

```diff
@@ -0,0 +1,18 @@
+# Digital Asset Management
+This sample illustrates how Computer Vision can add a layer of insights to a collection of images. 
+
+<p align="center">
+  <img src="ReadmeAssets/Screenshot.jpg" />
+</p>
+
+## Prerequisites
+
+* PC with Windows 10 version 17763 or higher
+* [Visual Studio 2017](https://visualstudio.microsoft.com/) or higher
+* Azure Subscription (you will need a [Cognitive Service](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesAllInOne) resource)
+
+## How it works
```

Comment:
We should add more high level info here about the code structure and how the file processing pipeline works. See the IoT Visual Alert one of an example.